EIPS/eip-increase-maxeb.md

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
```suggestion
---
eip: 7251
```
Issuing EIP number using PR number.  Please also update the filename.

Reply:
done!